generator/_assets/styles/less/variables.less

```diff
@@ -32,4 +32,4 @@
 @desktop-wide-down: ~'(max-width: 1439px)';
 @desktop-down: ~'(max-width: 1279px)';
 @tablet-down: ~'(max-width: 1023px)';
-@phone-down: ~'(max-width: 589px)';
+@phone-down: ~'(max-width: 589px)';
```

Comment:
please add new line to the end of the file

Reply:
fixed